ansible/roles/pg_standby/templates/create_postgres_dump.sh.j2

```diff
@@ -3,7 +3,10 @@ BACKUP_TYPE=$1
 DAYS_TO_RETAIN_BACKUPS=$2
 
 TODAY=$(date +"%Y_%m_%d")
-pg_basebackup -D "{{ postgresql_backup_dir }}/postgres_${BACKUP_TYPE}_${TODAY}" -Ft -Xs -z
+DIRECTORY="{{ postgresql_backup_dir }}/postgres_${BACKUP_TYPE}_${TODAY}"
+pg_basebackup -D $DIRECTORY -Xs
+tar -cjf "${DIRECTORY}.tar.bz" -C "${DIRECTORY}" .
+rm -rf $DIRECTORY
```

Comment:
The backup process now does a filesystem level dump that is then compressed, and the original dump is removed. This does mean we always need as much free space as the size of the db, but the db would have to more than double its current size before this starts to become a problem. I switched from gzip to bzip2 because it gives a higher compression ratio and while it is slower, the speed seemed less important.